jupyter-book/preprocessing_visualization/preprocessing.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    adata = sc.read('cellranger_neurips21_bmmc.h5ad')
</pre><p> </p><p>Can we add the dataset to figshare, please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
haven't added it yet before we confirmed with Chris that this adjusted preprocessing is still reasonable, should we nevertheless add it?

Reply:
No. Good point!